app/styles/ui/_merge-status.scss

```
@@ -17,7 +17,7 @@
 
     &:after {
       display: block;
-      border-bottom: var(--base-border);
+      border-bottom: 0;
```

We still need the horizontal rule here so it shows up in the comparison panel 😄 

```suggestion
      border-bottom: var(--base-border);
```

---

Ah, I must of left this off by accident, thank you so much! And now I get to test out the new 'apply suggestion' feature 👍